src/dstack/_internal/cli/commands/pool.py

```diff
@@ -0,0 +1,396 @@
+import argparse
+from pathlib import Path
+from typing import Sequence
+
+from rich.table import Table
+
+from dstack._internal.cli.commands import APIBaseCommand
+from dstack._internal.cli.services.configurators.profile import (
+    apply_profile_args,
+    register_profile_args,
+)
+from dstack._internal.cli.utils.common import confirm_ask, console
+from dstack._internal.core.errors import CLIError, ServerClientError
+from dstack._internal.core.models.instances import (
+    InstanceAvailability,
+    InstanceOfferWithAvailability,
+)
+from dstack._internal.core.models.pools import Instance, Pool
+from dstack._internal.core.models.profiles import Profile, SpotPolicy, TerminationPolicy
+from dstack._internal.core.models.resources import DEFAULT_CPU_COUNT, DEFAULT_MEMORY_SIZE
+from dstack._internal.core.models.runs import Requirements
+from dstack._internal.core.services.configs import ConfigManager
+from dstack._internal.utils.common import pretty_date
+from dstack._internal.utils.logging import get_logger
+from dstack.api._public.resources import Resources
+from dstack.api.utils import load_profile
+
+logger = get_logger(__name__)
+
+
+class PoolCommand(APIBaseCommand):  # type: ignore[misc]
```

Comment:
Remove `# type: ignore` comments since we're not enforcing type checking. Type checking should be a separate issue if to be introduced.

Reply:
Deleted all `# type: ignore` comment.